azavea/iow-boundary-tool
# Create project skeleton and update library versions

## Overview

* This PR contains the initial setup of the project from the [Civic Apps App Template](https://github.com/azavea/civic-apps-app-template).
* Some library versions in the template have since been updated, so I've updated all libraries and dependencies to the newest version except React. The current version of React (18) was just released on March 29th, so it was elected to leave React 17.0.2 for this project

Connects #1 

### Demo

### Notes

## Testing Instructions
* clone the project
* run `/scripts/update`
* run `/scripts/server` and ensure that it starts
* inspect the following files and ensure that they appear correct
`src/app/package.json`
`docker-compose.yml`
